0004/README.md

```diff
@@ -0,0 +1,416 @@
+# Proposal 4: SLSA Conformance Program
+* Proposers: [Kris K](https://github.com/kpk47) (kkris@google.com), [Joshua Mulliken](https://github.com/JoshuaMulliken) (jmullike@redhat.com)
+* GitHub issue: https://github.com/slsa-framework/slsa/issues/515
+* Status: [DRAFT](../README.md#meaning-of-status-codes)
+
+## Abstract
+
+This document describes a mechanism for labeling build platforms as officially
+SLSA conformant, the "SLSA Conformance Program." This proposal lays out the
+program at a high level, including a draft of the survey build platforms would
+need to fill out to participate. Please focus review on the program as a whole
+and the survey's general contents, reserving discussion of the survey's final
+wording for implementation PRs.
+
+## Problem statement
+
+One of SLSA's
+[guiding principles](https://slsa.dev/spec/v1.0/principles#trust-platforms-verify-artifacts)
+is to "Trust platforms, verify artifacts." The SLSA v1.0 specification reflects
+this principle's asymmetry through how it treats [verifying build
+platforms](https://slsa.dev/spec/v1.0/verifying-systems) and [verifying
+artifacts](https://slsa.dev/spec/v1.0/verifying-artifacts) – the former is a
+list of freeform questions a verifier might wish to ask of a build system, and
+the latter is a detailed procedure to be implemented in a tool. It is difficult
+and expensive to answer the question "is this build platform SLSA conformant?"  
```

Comment:
Please briefly explain when this may or may not be the case. For example, anyone can easily answer the question whether a build platform produces any signed/unsigned provenance, but not whether it is isolated and hosted.